server/index.js

```diff
@@ -1,10 +1,12 @@
 const express = require('express')
+const cors = require('cors')
 
 const app = express()
 
 const connectDB = require('./db/mongoose');
 const profileRouter = require('../server/router/profile')
 app.use(express.json());
+app.use(cors())
```

Comment:
I see that we had to use cors since it's the same host. Makes sense since he is okay with us running on our localhost machines